app/main.py

```diff
@@ -1,3 +1,85 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+from app import parsing
+
+import datetime
+
+
+def shop_trip() -> None:
+    config = parsing.parse_data_from_json()
+    customers_data = config["customers"]
+    shops_data = config["shops"]
+
+    customers = []
+    shops = []
+
+    for customer_data in customers_data:
+        car_data = customer_data["car"]
+        car = Car(car_data["brand"], car_data["fuel_consumption"])
+        customer = Customer(
+            customer_data["name"],
+            customer_data["location"],
+            customer_data["money"],
+            customer_data["product_cart"],
+            car
+        )
+        customers.append(customer)
+
+    for shop_data in shops_data:
+        shop = Shop(
+            shop_data["name"],
+            shop_data["location"],
+            shop_data["products"]
+        )
+        shops.append(shop)
```

Comment:
use a list comprehension

Reply:
not fixed. Create a list of Shop objects from shops_data using list comprehension